src/course-about/CourseAboutPage.scss

```diff
@@ -0,0 +1,23 @@
+@import "course-intro/course-media/CourseMedia";
+
+.course-about-intro {
+  .pgn__card {
+    min-height: var(--catalog-course-about-page-intro-card-min-height, 15.75rem);
```

Comment:
The logic around `min-height` here feels a bit odd.

I understand wanting to see

<img width="1500" height="311" alt="image" src="https://github.com/user-attachments/assets/f48a030c-9bde-4dcc-896d-adde3d160718" />

instead of

<img width="1500" height="311" alt="image" src="https://github.com/user-attachments/assets/6f5f756d-ac8b-4e07-95b8-ada930529ad0" />

but once a course actually has a description we end up with the opposite issue

realistic:

<img width="1500" height="311" alt="image" src="https://github.com/user-attachments/assets/fb2ea5fb-2f0a-4bd9-8e45-0ca21954b8d3" />

overboard:

<img width="1500" height="560" alt="image" src="https://github.com/user-attachments/assets/cc6af4f2-0efb-40d6-8129-29d406bb2ad2" />